<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaxesExcelFileCfgForm</class>
 <widget class="QDialog" name="TaxesExcelFileCfgForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3,0">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>主识别单元</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>纳税人识别码</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QLineEdit" name="edtNsrName"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>纳税人名称</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>发票所属日期</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="edtEnd"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edtNsrCode"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="edtStart"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>扫描区设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="horizontalSpacing">
       <number>30</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="1" column="6">
       <widget class="QLabel" name="lblDate">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>发票日期</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="LetterSpinBox" name="spnDate"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="lblMoney">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>发票金额</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="LetterSpinBox" name="spnMoney"/>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>作废标志</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblTaxMoney">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>税额</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="LetterSpinBox" name="spnGfCode"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>发票代码</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" colspan="3">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>购方纳税人识别码</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LetterSpinBox" name="spnNumber"/>
      </item>
      <item row="1" column="3">
       <widget class="LetterSpinBox" name="spnInvoiceCode"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>开始列</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="LetterSpinBox" name="spnInvoiceNumber"/>
      </item>
      <item row="2" column="5">
       <widget class="LetterSpinBox" name="spnRepealTag"/>
      </item>
      <item row="2" column="6">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>发票属性</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="LetterSpinBox" name="spnType"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblStartRow">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>开始行</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="lblInvoiceNumber">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>发票号码</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="LetterSpinBox" name="spnTaxMoney"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblNumber">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>序号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spnStartRow">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="LetterSpinBox" name="spnStartCol">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,1,0,0,0">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>当前配置名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbCfgs"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveAs">
       <property name="text">
        <string>保存当前配置为...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LetterSpinBox</class>
   <extends>QSpinBox</extends>
   <header>widgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>TaxesExcelFileCfgForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>TaxesExcelFileCfgForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>672</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>734</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
